projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-step/lab-step.component.ts

```diff
@@ -58,8 +69,28 @@ export class LabStepComponent implements OnInit, OnDestroy, AfterViewChecked {
 
   }
 
-  ngAfterViewChecked() {
-    if (!this.codeHighlighted) {
+  ngAfterViewChecked() 
+  {
+
+  if(!this.TriggerDialogB)
+  {
+    if(this.content){
+      let ImagesList = this.content.nativeElement.querySelectorAll("img");
+      for(var i=0;i<ImagesList.length;i++)
```

Comment:
use let instead of var, because let has local scope, var will set things globally
also, naming convention for imagesList